.pre-commit-config.yaml

```diff
@@ -1,29 +1,9 @@
-# default pre-commit file, checks node.js code and basic file formatting
+# pre-commit for deployment related resources
+# e.g. shell files, dockerfiles, helm chart etc..
 repos:
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: 'v8.19.0' # Use the sha / tag you want to point at
+  - repo: https://github.com/hadolint/hadolint
+    rev: 'v2.10.0'
     hooks:
-      - id: eslint
-        types: [file]
-        files: \.[jt]s$|vue$ # *.js, *.ts and vue
-        exclude: '(\/|^)((generated\/.*)|(\..*\.([jt]sx?|vue)))$'
-        args:
-          - '--max-warnings=0'
-        additional_dependencies:
-          - eslint@8.11.0
-          - eslint-config-prettier@8.5.0
-          - eslint-plugin-vue@8.5.0
-          - '@babel/eslint-parser@7.18.2'
-          - '@babel/preset-env@t 7.16.11'
-          - '@typescript-eslint/eslint-plugin@5.21.0'
-          - '@typescript-eslint/parser@5.21.0'
-          - typescript@4.5.4
-          - '@rushstack/eslint-patch@1.1.3'
-          - '@vue/eslint-config-typescript@11.0.0'
-
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: 'v2.7.1' # Use the sha / tag you want to point at
-    hooks:
-      - id: prettier
+      - id: hadolint
```

Comment:
I think we also need shellcheck & helm lint from the other pre-commit config file.

Reply:
yup, ur right